frontend/src/Utils/useActiveNamespace.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { isUtilsConfigSet, k8sListResourceItems } from '@openshift/dynamic-plugin-sdk-utils';
+import { ProjectModel } from './types';
+
+/**
+ * Hook to calculate active namespace from ProjectModel.
+ * @returns active namespace, if found.
+ */
+const useActiveNamespace = () => {
```

Comment:
Looks like this code isn't used.

Reply:
Yeah, we thinking if we should keep it or not. It could be usefull for someone in the future. But let's remove it, so we don't have unused code.